lib/validate.js

```diff
@@ -130,7 +130,7 @@ function checkSubSchema(schema, parent, state) {
     }
     if (schema.pattern) {
         try {
-            let regex = new RegExp(schema.pattern);
+            const _ = new RegExp(schema.pattern);
```

Comment:
Is the throwaway `_` to get around a different lint error?  Unused `regex` perhaps?

This is awkward

Reply:
Yeah it's awkward. This was showing a lint moan, and if I remove it it complains about "do not use new for side effects." 

Again I'm just twiddling bits as I go. 🤷‍♂️